kafka_consumer/pyproject.toml

```diff
@@ -38,6 +38,7 @@ text = "BSD-3-Clause"
 
 [project.optional-dependencies]
 deps = [
+    "confluent-kafka==2.0.2; python_version > '3.0'",
```

Comment:
nit - could we add a comment to explain that this dependency is in fact built in omnibus and bumping it here will have no effect (just like snowflake)?

Reply:
Added!